src/ceph-volume/ceph_volume/devices/simple/scan.py

```diff
@@ -263,6 +266,15 @@ def scan(self, args):
             with open(json_path, 'w') as fp:
                 json.dump(osd_metadata, fp, indent=4, sort_keys=True, ensure_ascii=False)
                 fp.write(os.linesep)
+            try:
+                os.chown(json_path, args.owner, args.group)
+                os.chmod(json_path, stat.S_IRUSR)  # 0400
+            except OSError as e:
+                msg = "Failed to set owner/group/permissions of %s: %s" % (
+                    json_path, str(e))
+                logger.warning(msg)
```

Comment:
this would benefit from seeing the actual exception
```suggestion
                logger.exception(msg)
```